lib/dropbox/client.rb

```diff
@@ -125,9 +125,23 @@ def get_thumbnail(path, format='jpeg', size='w64h64')
     # Get the contents of a folder.
     #
     # @param [String] path
+    # @param [boolean] recursive
+    # @param [boolean] include_media_info
+    # @param [boolean] include_deleted
+    # @param [boolean] include_has_explicit_shared_members
     # @return [Array<Dropbox::Metadata>]
-    def list_folder(path)
-      resp = request('/files/list_folder', path: path)
+    def list_folder(path,
+      recursive = false,
+      include_media_info = false,
+      include_deleted = false,
+      include_has_explicit_shared_members = false	)
```

Comment:
+1 as this feels more ruby-esque

and *bump* as I'm in need of the recursive option right now. =P